types/jsreport-core/index.d.ts

```diff
@@ -0,0 +1,117 @@
+// Type definitions for jsreport-core 1.5
+// Project: http://jsreport.net
+// Definitions by: taoqf <https://github.com/taoqf>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+/// <reference types="node" />
+
+declare namespace JsReport {
+  type Helpers = string | { [fun: string]: (...args: any[]) => any };
+
+  const enum EngineType {
+    None = "none"
+  }
+
+  const enum RecipeType {
+    // Html = "html"
```

Comment:
Why is this removed? There is html recipe right in the jsreport core [here](https://github.com/jsreport/jsreport-core/blob/master/lib/render/htmlRecipe.js).

Reply:
Sorry, I didn't know that.